pkg/kubectl/cmd/get.go

```diff
@@ -464,6 +470,10 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 				filteredResourceCount++
 				continue
 			}
+			if errs.Has(err.Error()) {
+				continue
```

Comment:
This if statement changes behavior -- previously, errors from filterfuncs would not block printerobj handling.

I think you want:

    if !errs.Has(err.Error()) {
        errs.Insert(err.Error())
        allErrs = append(allErrs, err)
    }